trie/verkle.go

```diff
@@ -268,7 +268,7 @@ func (t *VerkleTrie) Commit(_ bool) (common.Hash, *trienode.NodeSet) {
 	nodeset := trienode.NewNodeSet(common.Hash{})
 	for _, node := range nodes {
 		// Hash parameter is not used in pathdb
-		nodeset.AddNode(node.Path, trienode.New(common.Hash{}, node.SerializedBytes))
```

Comment:
this should be non-zero

Reply:
Ok, this is besides the point as verkle isn't used for bloatnet. Let's leave it at that. Just fix it in the PR against master please.

Reply:
nvm I had it all wrong. Yes it's incorrect for verkle but it's not easy to get for an internal node. Not to worry at this time.